### PR TITLE
LPS-200205 Remove Saved content Entries when deleting Asset entries or Users

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
How to test it:

- Activate FF (LPS-197909)
- edit Home Page
- add Asset Publisher widget
- select dynamic > Document
- Publish
- create Display Page template for basic documents with new Saved Content taglib and mark as default
- Create new document

For asset entry (docs on this example)
- Go to home
- See created document > save content (new register should be created on database)
- delete created document (from recycle bin too)
- the register on database should not longer be present

For user
- Create new user and add it as site member
- impersonate it
- Go to home
- See created document > save content (new register should be created on database with the new user)
- remove user (first deactivate, after filter by deactivated, and remove the created user)
- the register on database should not longer be present

### CODE REVIEW

File: modules/apps/saved-content/saved-content-service/src/main/java/com/liferay/saved/content/service/impl/SavedContentEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -103,6 +104,14 @@ public SavedContentEntry fetchSavedContentEntry(
 			classPK);
 	}
 
+	@Override
+	public List<SavedContentEntry> findSavedContentEntries(
+		long groupId, String className, long classPK) {
+
+		return savedContentEntryPersistence.findByG_C_C(
+			groupId, _classNameLocalService.getClassNameId(className), classPK);
+	}
```
Comment (adolfopa) [inline modules/apps/saved-content/saved-content-service/src/main/java/com/liferay/saved/content/service/impl/SavedContentEntryLocalServiceImpl.java:113]: Why not create a `deleteSavedContentEntries` method instead? Build service should have created a `removeByG_C_` that you can call directly. This way, you avoid the need to loop in the model listener.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/saved-content/saved-content-service/src/main/java/com/liferay/saved/content/service/impl/SavedContentEntryLocalServiceImpl.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/saved-content/saved-content-service/src/main/java/com/liferay/saved/content/service/impl/SavedContentEntryLocalServiceImpl.java]: great, thanks!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1387401_9698

**Comment by WYZ-Issac** [comment]
Failures are unrelated.

**Comment by WYZ-Issac** [comment]
ci:forward:force

**Pull request closed without merging**